From ae46b45c48472585bb0040bd6919642928e82cf3 Mon Sep 17 00:00:00 2001
From: Bo Shen <voice.shen@atmel.com>
Date: Fri, 6 Mar 2015 16:56:52 +0800
Subject: [PATCH] Board: at91sam9rlek: remove allow 266MHz

As the at91sam9rl SoC can not work more than 200MHz. So, work
at 266MHz is not allowed, remove it.

Signed-off-by: Bo Shen <voice.shen@atmel.com>
Acked-by: Nicolas Ferre <nicolas.ferre@atmel.com>
---
 board/at91sam9rlek/Config.in.board | 1 -
 1 file changed, 1 deletion(-)

diff --git a/board/at91sam9rlek/Config.in.board b/board/at91sam9rlek/Config.in.board
index 94d1413..dc61640 100644
--- a/board/at91sam9rlek/Config.in.board
+++ b/board/at91sam9rlek/Config.in.board
@@ -6,7 +6,6 @@ config	CONFIG_AT91SAM9RLEK
 	select ALLOW_SDCARD
 	select CONFIG_SDRAM
 	select ALLOW_CPU_CLK_200MHZ
-	select ALLOW_CPU_CLK_266MHZ
 	select ALLOW_CRYSTAL_18_432MHZ
 	select ALLOW_BOOT_FROM_DATAFLASH_CS0
 	select ALLOW_DATAFLASH_RECOVERY
-- 
1.9.1

